### ISSUE TITLE
Print prefix for s.SuperType

### ISSUE THREAD

**Issue by olafurpg** [scalacenter/scalafix#758]
Fix the following TODO https://github.com/scalacenter/scalafix/blob/264a8ffc15e74c5ce994d913786bb9ffd4ea5929/scalafix-core/shared/src/main/scala/scalafix/internal/util/PrettyType.scala#L562

Can you give an example input for this case @xeno-by ?

**Comment by xeno-by** [comment]
```
scala> class X
defined class X

scala> class C { val x = new X }
defined class C

scala> class D extends C { override val x = new X; type T = super.x.type }
defined class D

scala> typeOf[D].decl(TypeName("T"))
res6: $r.intp.global.Symbol = type T

scala> showRaw(res6.info)
res7: String = SingleType(SuperType(ThisType(D), TypeRef(SingleType(SingleType(SingleType(SingleType(SingleType(SingleType(SingleType(SingleType(SingleType(SingleType(SingleType(SingleType(SingleType(SingleType(SingleType(SingleType(SingleType(SingleType(SingleType(SingleType(ThisType(<root>), $line25), $line25.$read), $iw), $iw), $iw), $iw), $iw), $iw), $iw), $iw), $iw), $iw), $iw), $iw), $iw), $iw), $iw), $iw), $iw), $iw), C, List())), TermName("x"))
```

**Comment by olafurpg** [comment]
This is most likely fixed in #989
